File: Unidad 4/P10_u5.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p10_u5;
import java.util.Random;
/**
 *
 * @author LaboratorioLTI2
 */
public class P10_u5 {

    public static void main(String[] args) {
        int longitud=8;
        String contrasena = aleatorio(longitud);
        System.out.println("password"+contrasena);
      
    
    }
    public static String aleatorio(int longitud){
     String caracteres = "ABCDEFGHIJK123456789*-_.";
     Random random = new Random();
     StringBuilder cadena= new StringBuilder(longitud);
     for(int i=0;i< longitud; i++){
     int indiceCaracter = random.nextInt(caracteres.length());
     cadena.append(caracteres.charAt(indiceCaracter));
     } 
     return cadena.toString();
    }
}

File: unidad5/p10_u5.cs

using System.Text;

namespace ProgramaU5_10;

class Password
{
    static void Main(string[] args)
    {
        int longitud = 8;
        Console.Write("Password: " + random(longitud));
    }
    public static String random(int x)
    { //x es el tamaño    
        //Caracteres
        string letras = "ABSLHDLHKAH/$%#&_jkjsaloqn"; 
        StringBuilder cadena = new StringBuilder(x); 
        Random alea = new Random(); 

        for (int i = 0; i < x; i++)
        {
           
            int IndiceCaracter = alea.Next(letras.Length); 
            cadena.Append(letras[IndiceCaracter]);

        }
        return cadena.ToString(); 
    }
}
